src/store/modules/Logs/DeconfigurationRecordsStore.js

```diff
@@ -80,7 +79,7 @@ const DeconfigurationRecordsStore = {
                 srcDetails: AdditionalData?.EventId,
                 status: AdditionalData?.Resolved, //true or false
                 uri: log['@odata.id'],
-                severity: Severity,
+                severity: MessageArgs[0],
```

Comment:
Severity is not populating in the records section
<img width="1726" alt="Screenshot 2024-11-05 at 11 09 38 AM" src="https://github.com/user-attachments/assets/a020b888-4c1d-4c14-9223-82b947b62f1d">
<img width="1717" alt="Screenshot 2024-11-05 at 11 10 22 AM" src="https://github.com/user-attachments/assets/a8b4cbc3-4558-44d9-87dd-8878de93ba65">

Reply:
Made the required changes